README.md

```diff
@@ -142,11 +154,11 @@ Then disable GNOME Power Profiles Daemon by runing:
 
 ### 2: `--force` governor override
 
-By default auto-cpufreq uses `balanced` mode which works the best on various systems and situations.
+By default auto-cpufreq uses `balanced` mode which works the best on various systems and situations. 
 
 However, you can override this behaviour by switching to `performance` or `powersave` mode manually. Performance will result in higher frequencies by default, but also results in higher energy use (battery consumption) and should be used if max performance is necessary. Otherwise `powersave` will do the opposite and extend the battery life to its maximum.
 
-See [`--force` flag](#overriding-governor) for more info.
+See [`--force` flag](https://github.com/AdnanHodzic/auto-cpufreq/#overriding-governor) for more info.
```

Comment:
Let's be consistent and update `See [`--force` flag](https://github.com/AdnanHodzic/auto-cpufreq/#overriding-governor) for more info` to new self referencing README structure i.e: (#overriding-governor))

Reply:
done